frontend/src/modules/lobby/components/Nickname.vue

```diff
@@ -1,10 +1,10 @@
 <template>
-  <div>
-    <b-button v-b-modal.modal-2>Nickname</b-button>
+  <span>
```

Comment:
Just curious, any reason for this being a span vs div?

Reply:
its so the button is in the same line